install/kubernetes/helm/istio/charts/grafana/templates/pvc.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.persist }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: istio-grafana-pvc
+  labels:
+    app: {{ template "grafana.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  storageClassName: {{ .Values.storageClassName }}
+  accessModes:
+    - ReadWriteOnce
```

Comment:
ReadWriteMany may solve the grafana replica problem.  could you try it?  If your PV storage doesn't support readwritemany, could always make accessModes configurable.

Cheers
-steve

Reply:
#8416